docs/deploy_infoblox.md

```diff
@@ -36,6 +35,18 @@ kubectl create ns k8gb
 make infoblox-secret
 ```
 
+* Configure NGINX Ingres Controller in order to expose DNS UDP port for `k8gb-coredns` service on working nodes.
```

Comment:
These recommendations are not isolated to Infoblox only. They should be also provided for a [Quickstart](https://www.k8gb.io/#quick-start) and also for a [local setup](https://www.k8gb.io/docs/local.html) in-depth doc.
In case of local setup we might just incorporate config map patching/creating within the associated Make target

Reply:
@ytsarev, good point, will amend make targets 👍

Reply:
@ytsarev, amended. Also created DNS configuration document.